go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/hashicorp/terraform-provider-hcs
 
-go 1.14
+go 1.15
```

Comment:
`1.15`? Should this be `1.16`?

Reply:
The go mod specifies the minimum Go version required to run the provider. Since the min version of the terraform-plugin-sdk is 1.15, that seems like the only compatibility we need to maintain.